po/de.po

```diff
@@ -19474,7 +19473,7 @@ msgstr "Keine Hauptperson"
 msgid ""
 "You need to set a 'default person' to go to. Select the People View, select "
 "the person you want as 'Home Person', then confirm your choice via the menu "
-"Edit ->Set Home Person."
+"Edit -> Set Home Person."
```

Comment:
correcting the original text makes no sence is generated from the sorce code

Reply:
Thanks for the info. In this case I have to correct that in the source code.

Reply:
You did the same thing at line 1708.  I think it would be best to NOT change this in the source code, as that would invalidate all other translations.  (Unless you want to change the pot and all the translations, in which case you should submit a separate PR for that.)

Reply:
I did change it in the source code AND all translations in this PR:
https://github.com/gramps-project/gramps/pull/756